src/libstd/sys/unix/c.rs

```diff
@@ -15,11 +15,26 @@
 
 pub use self::select::fd_set;
 pub use self::signal::{sigaction, siginfo, sigset_t};
+#[cfg(not(target_os = "nacl"))]
 pub use self::signal::{SA_ONSTACK, SA_RESTART, SA_RESETHAND, SA_NOCLDSTOP};
```

Comment:
@alexcrichton despite globs being stable, the general convention being followed is to avoid them in much the same way and for much the same reasons as they’re advised against in PEP 8.